src/main/java/space/obminyashka/items_exchange/controller/error/GlobalExceptionHandler.java

```diff
@@ -101,6 +101,18 @@ public ErrorMessage handleIOException(Exception ex, ServletWebRequest request){
         return logAndGetErrorMessage(request, ex, Level.ERROR);
     }
 
+    @ExceptionHandler({EmailTokenNotExistsException.class})
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
You should check first of all whether the handler has a handler method for such status and simply add that Exception to that handler, no need to duplicate it for every case.